znai-docs/documentation/meta.json

```diff
@@ -4,5 +4,8 @@
   "viewOn": {
     "link": "https://github.com/twosigma/znai/blob/master/znai-docs/documentation",
     "title": "View On GitHub"
+  },
+  "support": {
+    "link": "https://twosigma.slack.com/archives/CEHLLNLMV"
```

Comment:
This is a tricky one to put here since for open sourced deployed docs we need a different URL

Reply:
deleted.